### PR TITLE
Awesome books

### PR DESCRIPTION
**Pull request by mmatongo**
### This is the Microverse module two week one project "Awesome Books".

This pull requests implements the following;
- Implement only a basic UI with plain HTML
- Create a collection that keeps a list of books
- Create a function to add a new book to the collection, with title and author.
- Create a function to remove a book from the collection
- Display all books saved in the collection in the top part of the page.
- Make sure that when a user clicks the "Add" button:
    - A new book is added to the collection.
    - The new book is displayed in the page.
- Make sure that when a user clicks the "Remove" button:
    - The correct book is removed from the collection.
    - The correct book disappears from the page.
- Make sure that data is preserved in the browser's memory by using localStorage.

### CODE REVIEW

**Review by mikenath223 — CHANGES_REQUESTED** [review]
Hello Team,

You've done pretty well so far!
But for a few issues, you are almost done with this. Hang in there!

## Required Changes ♻️
_Please check the comments under the review._
- [x] Change regular functions to es6 arrow functions.

Happy coding, Cheers! 🚀🚀

Do feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, whether it is valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: js/main.js [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+window.onload = () => {
+  if (localStorage.getItem('Books')) {
+    const oldStorage = localStorage.getItem('Books');
+    const newStorage = JSON.parse(oldStorage);
+    books = newStorage;
+    genHTML();
+  } else {
+    genHTML();
+  }
+};
```
Comment (mikenath223) [inline js/main.js:42]: - [ ] Kindly change these functions to es6 arrow functions, as required under the [JavaScript best practices guide](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/javascript_best_practices.md#:~:text=If%20you%20use%20ES6%2C%20use%20object%20destructuring%20to%20get%20the%20values%20from%20an%20object.%20This%20way%20you%20can%20avoid%20repeating%20a%20lot%20of%20code.) to use es6 syntaxes.

**Comment by rpire** [comment]
Hello @mikenath223 👋

Thank you for reviewing our work. We will make sure to implement the requested changes before merging. 🤓

Cheers and Happy Coding! 🚀

File: js/main.js (hunk 1 above)
Comment (rpire) [reply js/main.js]: Requested changes completed. 🤓

**Comment by rpire** [comment]
## Changes made:

- [x] Changed all regular functions to arrow functions according to the requested changes.

**Comment by Davidosky007** [comment]
## Hi @rpire 👋 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
## Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Comment by rpire** [comment]
Hi @Davidosky007 👋

# Thank you! 🙌

I'll proceed to merge.

Cheers and Happy Coding. 🚀

**Pull request merged**